index.html

```diff
@@ -39,7 +39,124 @@ <h2 class="caption">I’am a Software Developer</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+        </section>
+    </section>
+    <section class="about-me">
+        <div class="me-content">
+            <div class="title"><h2>About Me</h2></div>
```

Comment:
- [ ] You have an overflow of text from this title. Give this a fix.
![Screenshot from 2021-08-14 17-18-59](https://user-images.githubusercontent.com/65501535/129449288-60d4d569-fc32-4f0b-bdf9-2cd8090d16cd.png)